src/styles/index.scss

```diff
@@ -1,3 +1,91 @@
+@import '/src/styles/card';
+@import '/src/styles/variables';
+@import '/src/styles/stars';
+@import '/src/styles/mixin';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
+
+  @include max-width;
+  @include min-width;
```

Comment:
The mixins `max-width` and `min-width` are used here. Ensure these mixins are defined in the imported `mixin` file, otherwise, it will lead to a compilation error.